autopush/tests/test_websocket.py

```diff
@@ -54,6 +54,7 @@ def setUp(self):
         self.proto.sendClose = self.close_mock = Mock()
         self.proto.transport = self.transport_mock = Mock()
         self.proto.closeHandshakeTimeout = 0
+        self.proto.autoPingInterval = 300
```

Comment:
Ben and I talked about this a bit more yesterday, but I'd love to hear your thoughts, too. A counter would satisfy our curiosity, but it's not really actionable. (This is the same guidance I've received for Sync telemetry in Firefox).

In our case, we already know clients are susceptible to this bug; we just can't do anything except institute the workaround. It doesn't tell us if the workaround is effective, just that clients are buggy. Reduced reconnect rates might be a better measure.

Ben pointed out this applies to other metrics we record, too—for example, what new information do we get from counting the number of client pings?